app/scripts/modules/core/src/pipeline/config/stages/script/scriptExecutionDetails.html

```diff
@@ -19,7 +19,7 @@
       </div>
     </div>
 
-    <div class="row ng-scope" ng-if="stage.context.buildInfo.url">
+    <div class="row ng-scope" ng-if="!ctrl.scriptRanAndFailed && stage.context.buildInfo.url">
```

Comment:
this mix of styles doesn't feel awesome.  I noticed BaseExecutionDetailsCtrl has 40+ usages so I get why.